workflow-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -0,0 +1,10 @@
+def branches = [:]
+for (int i = 0; i < 4; i++) {
+  def which = "${i}" 
```

Comment:
see comments. below
since this specific use case is for worker jobs, they can run with identical configuration and don't need any mechanism to distinguish between each build it's perfectly fine to use a dummy parameter, imho, because you don't care about differentiation queue entries.